partials/forms/radio-group.html

```diff
@@ -1,4 +1,5 @@
-<div id="{{key}}-group" class="form-group{{#display}} {{display}}{{/display}}{{#error}} validation-error{{/error}}">
+<fieldset id="{{key}}-group" class="{{#display}}{{display}}{{/display}}{{#error}} validation-error{{/error}}">
+    <legend{{#legend-display}} class="{{legend-display}}"{{/legend-display}}>{{legend}}</legend>
```

Comment:
Is there a case yet for this to be variable/configurable? I'd be tempted to hard-code it if it's going to be the same everywhere.